frontend/islands/HomepageHero.tsx

```diff
@@ -152,7 +152,7 @@ export function HomepageHero(
         <script src={asset("/scripts/particles.js")} defer></script>
       </Head>
       <style>{canvasStyle}</style>
-      <div class="section-x-inset-xl flex flex-col items-center justify-center gap-12 relative">
+      <div class="section-x-inset-xl flex flex-col items-center justify-center gap-12 relative pointer-events-none">
```

Comment:
I'm not so sure about this one. I understand disabling select for interactive controls (and it is nice to have the background interactivity here), but this is just a heading, and a user might legitimately want to select the text. I'd prefer to err on the side of trusting the user here.

Reply:
This `pointer-events-none` is not for disabling selectability. Users can still select the text in the header. This lets the user interact with the animation (or whatever it is called) from empty parts of the header.

Reply:
Ah yes I see that; my mistake.